### PR TITLE
KIALI-1748 Updating features visuals on landing page

### PR DESCRIPTION
**Pull request by abonas**
featuresbox

### CODE REVIEW

**Comment by abonas** [comment]
@theute some images might be refitted better later by our designer, but I see no point blocking the PR on this, we can always reupload them.

**Comment by theute** [comment]
Visually, I liked it better when the box was overlapping the blue backgroun, was it a request from UX to get rid of this ?
The metrics image isn't round just because the way it's cropped could we did a quick fix here before merging ?

**Comment by abonas** [comment]
@aljesusg @mtho11 any idea how to make the features box to be half on blue and half on white like @theute suggests?
the design is indeed suggesting this but I'm not able to force it :)
screenshot at 2018-12-02 15-56-04

**Comment by aljesusg** [comment]
@abonas add    position: relative;  and  top: -50px; (Increase this to have the box more up). Other
```css
.features-bottom-box {
  display: inline-block;
  padding: 5px;
  top: -50px;
  position: relative; 
  background: #fff;
  box-shadow: 0 10px 40px 0 rgba(47, 47, 47, .1);
  border-radius: 3px;
  cursor: pointer
}
```

screenshot from 2018-12-03 08-30-07

To have the same consistency add width: 100%; to .features-bottom-box  and you need features-list . This is the result 

```css

.features-bottom-box {
  display: inline-block;
  padding: 5px;
  top: -50px;
  position: relative;
  width: 100%;
  background: #fff;
  box-shadow: 0 10px 40px 0 rgba(47, 47, 47, .1);
  border-radius: 3px;
  cursor: pointer
}

#features-list {
  display: flex;
  margin-right: 15px;
  margin-left: 15px;
  justify-content: space-between !important;
}
```

screenshot from 2018-12-03 08-35-14

**Comment by abonas** [comment]
@aljesusg many thanks!! I keep forgetting the 'position' option *all the time* (not the first time ;))

**Comment by abonas** [comment]
@aljesusg btw the width: 100%; is creating a lot of "dead" white space to the right of the feature visuals in the features white box, why did you add it? (if it's reduced to 76% for example, it looks better)

**Comment by aljesusg** [comment]
> why do you need to add this part?
> justify-content: space-between !important;

This part make the icons inside the box have the same space between them.

> @aljesusg btw the width: 100%; is creating a lot of "dead" white space to the right of the feature visuals in the features white box, why did you add it? (if it's reduced to 76% or completely removed, for example, it looks better)

I set the width 100% to have the max width that his parents div who have the same space defined in the dive where is the get started button. I don't know what you mean with dead space. Can you add a screenshot?

**Comment by abonas** [comment]
@theute better ^^ (and thanks to @aljesusg for helping positioning it better)

**Comment by aljesusg** [comment]
Anyway @abonas I am not an expert in css^^ but I can defend myself a little ^^

**Comment by theute** [comment]
Looks better, but still have 3 comments (can be fixed later if needed)
1 - Wouldn't it look better if the features box was horizontally centered ?

Ok, just 1, because the other 2 comments ust got magically fixed ;)

**Comment by abonas** [comment]
> Anyway @abonas I am not an expert in css^^ but I can defend myself a little ^^

me neither @aljesusg !

what I mean by dead space - see here to the right of the tracing feature visual - a lot of empty white space:
screenshot at 2018-12-03 15-00-01

**Comment by aljesusg** [comment]
> > Anyway @abonas I am not an expert in css^^ but I can defend myself a little ^^
> 
> me neither @aljesusg !
> 
> what I mean by dead space - see here to the right of the tracing feature visual - a lot of empty white space:
> screenshot at 2018-12-03 15-00-01

But with justify-content: space-between !important; you fix it

File: static/css/kiali_style.css [hunk 1]
```diff
@@ -64,17 +64,63 @@ p {
...
   box-shadow: 0 10px 40px 0 rgba(47, 47, 47, .1);
   border-radius: 3px;
   cursor: pointer
 }
 
+#features-list {
+  display: flex;
+  margin-right: 15px;
+  margin-left: 15px;
+  display: inline-block;
```
Comment (aljesusg) [inline static/css/kiali_style.css]: You must remove the lien 81. this is overwritting teh display:flex

File: static/css/kiali_style.css [hunk 2]
```diff
@@ -64,17 +64,63 @@ p {
 }
 
 .features-bottom-box {
-  display: block;
-  padding: 40px;
+  display: inline-block;
+  padding: 5px;
+  top: -50px;
+  position: relative;
   background: #fff;
```
Comment (aljesusg) [inline static/css/kiali_style.css]: add width: 100%

**Comment by abonas** [comment]
@aljesusg brilliant :) it works!

**Comment by abonas** [comment]
@theute I think that *now* it is better :)

**Comment by aljesusg** [comment]
Great anyway we need to add some css for this box for Mobile version
screenshot_20181203-160130__01

**Comment by abonas** [comment]
@theute merge please?

**Comment by theute** [comment]
It is really broken on a mobile, do we really want to get this in before fixing this ? 
(Right now kiali.io works great on mobile)

**Comment by abonas** [comment]
@aljesusg I don't see that the proposed above fix fixes the mobile, I checked both the netlify uploaded version and with the mobile simulator built in in Chrome.
I bet I'm doing something wrong...

**Comment by abonas** [comment]
@theute are you seeing with this addition that Alberto proposed that mobile is fixed? I don't, so I am wondering if we are checking it in different ways

File: static/css/kiali_style.css [hunk 3]
```diff
@@ -64,17 +64,69 @@ p {
...
   border-radius: 3px;
   cursor: pointer
 }
 
+@media screen and (max-width: 768px) {
+  #features-list  {
+    display: inline-block;
+  }
+}
+
```
Comment (aljesusg) [inline static/css/kiali_style.css]: You need to move this down the #features-list, this is compilers from top to down so L84-L89 overwrite the display.

Comment (aljesusg) [reply static/css/kiali_style.css]: anyway I think that would be better set inline-grid or grid  instead inline-block

Comment (abonas) [reply static/css/kiali_style.css]: I tried all the variations, they all look the same to me. I pushed the code.

**Comment by theute** [comment]
I see 
image
on mobile.

PS: we have 26% of visitors on mobile

**Comment by abonas** [comment]
@theute latest :
screenshot_20181206-131058_chrome

**Pull request merged**